pkg/util/constants/pooler.go

```diff
@@ -0,0 +1,13 @@
+package constants
+
+// Connection pool specific constants
+const (
+	ConnectionPoolUserName             = "pooler"
+	ConnectionPoolSchemaName           = "pooler"
+	ConnectionPoolDefaultType          = "pgbouncer"
+	ConnectionPoolDefaultMode          = "transition"
+	ConnectionPoolDefaultCpuRequest    = "100m"
+	ConnectionPoolDefaultCpuLimit      = "100m"
+	ConnectionPoolDefaultMemoryRequest = "100M"
```

Comment:
`Mi`? or is there no difference?

Reply:
Well, it's M for decimal SI and Mi for binary SI. But everywhere else we do Mi, so we can change it here.